lib/doubly_linked_list.rb

```diff
@@ -0,0 +1,46 @@
+# doubly_linked_list.rb
```

Comment:
It's great that you started on the DoublyLinked list, but the tests aren't running, I'll just review this file for now as it looks like you were in the middle of doing it.